cloud/pkg/synccontroller/synccontroller.go

```diff
@@ -0,0 +1,252 @@
+	config, err := buildConfig()
+	if err != nil {
+		klog.Errorf("Failed to build config, err: %v", err)
+		os.Exit(1)
+	}
+	kubeClient := kubernetes.NewForConfigOrDie(config)
+	crdClient := versioned.NewForConfigOrDie(config)
+
+	kubeSharedInformers := informers.NewSharedInformerFactory(kubeClient, 0)
+	crdFactory := crdinformerfactory.NewSharedInformerFactory(crdClient, 0)
+
+	podInformer := kubeSharedInformers.Core().V1().Pods()
+	configMapInformer := kubeSharedInformers.Core().V1().ConfigMaps()
+	secretInformer := kubeSharedInformers.Core().V1().Secrets()
+	serviceInformer := kubeSharedInformers.Core().V1().Services()
+	endpointInformer := kubeSharedInformers.Core().V1().Endpoints()
+	nodeInformer := kubeSharedInformers.Core().V1().Nodes()
+	deviceInformer := crdFactory.Devices().V1alpha1().Devices()
+	clusterObjectSyncInformer := crdFactory.Reliablesyncs().V1alpha1().ClusterObjectSyncs()
+	objectSyncInformer := crdFactory.Reliablesyncs().V1alpha1().ObjectSyncs()
+
+	sctl := &SyncController{
+		podInformer:               podInformer,
+		configMapInformer:         configMapInformer,
+		secretInformer:            secretInformer,
+		serviceInformer:           serviceInformer,
+		endpointInformer:          endpointInformer,
+		nodeInformer:              nodeInformer,
+		deviceInformer:            deviceInformer,
+		clusterObjectSyncInformer: clusterObjectSyncInformer,
+		objectSyncInformer:        objectSyncInformer,
+
+		podSynced:               podInformer.Informer().HasSynced,
+		configMapSynced:         configMapInformer.Informer().HasSynced,
+		secretSynced:            secretInformer.Informer().HasSynced,
+		serviceSynced:           serviceInformer.Informer().HasSynced,
+		endpointSynced:          endpointInformer.Informer().HasSynced,
+		nodeSynced:              nodeInformer.Informer().HasSynced,
+		deviceSynced:            deviceInformer.Informer().HasSynced,
+		clusterObjectSyncSynced: clusterObjectSyncInformer.Informer().HasSynced,
+		objectSyncSynced:        objectSyncInformer.Informer().HasSynced,
+
+		podLister:               podInformer.Lister(),
+		configMapLister:         configMapInformer.Lister(),
+		secretLister:            secretInformer.Lister(),
+		serviceLister:           serviceInformer.Lister(),
+		endpointLister:          endpointInformer.Lister(),
+		nodeLister:              nodeInformer.Lister(),
+		clusterObjectSyncLister: clusterObjectSyncInformer.Lister(),
+		objectSyncLister:        objectSyncInformer.Lister(),
+	}
+
+	return sctl
+}
+
+func Register(ec *configv1alpha1.SyncController, kubeAPIConfig *configv1alpha1.KubeAPIConfig) {
+	config.InitConfigure(ec, kubeAPIConfig)
+	core.Register(newSyncController())
+}
+
+// Name of controller
+func (sctl *SyncController) Name() string {
+	return SyncControllerModuleName
+}
+
+// Group of controller
+func (sctl *SyncController) Group() string {
+	return SyncControllerModuleGroup
+}
+
+// Group of controller
+func (sctl *SyncController) Enable() bool {
+	return true
```

Comment:
add `Enable` field for `SyncController  struct` and then use sctl.Enable.

Reply:
Done.